socorro/cron/jobs/ftpscraper.py

```diff
@@ -333,7 +333,7 @@ def scrape_json_releases(self, connection, product_name):
         for directory in ('nightly', 'candidates'):
             try:
                 url, = self.get_links(prod_url, starts_with=directory)
-            except IndexError:
+            except (IndexError, ValueError):
```

Comment:
If the result of ``self.get_links`` is empty, the ``url, =`` thing cannot work and raises the ``ValueError``. As Peter said, I wonder what triggered the ``IndexError``, but I honestly didn't want to spend any time looking into that... :)